static/css/place/place.scss

```diff
@@ -347,4 +347,12 @@ h3 {
 .ranking-chart h4 {
   text-align: center;
   margin-top: 0rem;
+}
+
+.map-container {
+  display: flex;
+
+  .map {
+    flex-grow: 1;
+  }
```

Comment:
curious what this css fixes on place explorer?

Reply:
also curious there's no css update to tools/map

Reply:
since I extracted the legend drawing functionality out of the original map drawing function, I also need to have one div to draw the map in and one to draw the legend in. This does the formatting for the two divs. The tools/map was originally already following this pattern so no need for css changes.